contracts/core/primitives/feed/Feed.sol

```diff
@@ -270,4 +270,8 @@ contract Feed is IFeed, RuleBasedFeed, AccessControlled {
     function getExtraData(bytes32 key) external view override returns (bytes memory) {
         return Core.$storage().extraData[key];
     }
+
+    function getAuthorPostId(address author, uint256 localSequentialId) external view returns (uint256) {
+        return Core._generatePostId(author, localSequentialId);
```

Comment:
Actually, this is wrong, this is the Post ID, not the author local sequential one

Reply:
Not sure what this getter was